### ISSUE TITLE
inverse_transform() not implemented

### ISSUE THREAD

**Issue by jctaillandier** [IBM/AIF360#164]
I am running the disparate impact demo. The output is encoded data and columns, and there isn't any inverse_transform function implemented in the code, as is the often the case, or maybe I am missing something ?

I would need to have the old 15-16 columns for post processing analysis, and try to avoid doing the recovery manually!

**Comment by hoffmansc** [comment]
You're correct that there is no inverse_transform function available. However, the original data (`train` or `test` in the demo) is unchanged by the `fit_transform()` function so you can still use that. Because of how the algorithm works, I don't think this would even be possible since there may be a many-to-one mapping in `transform()`.
